.github/workflows/main.yml

```diff
@@ -44,6 +44,7 @@ jobs:
         pushd _benchmarks_snapshot
         7z x "$tmpfile"
         popd
+        .github/scripts/run-migration.sh
```

Comment:
As the directory names like `_benchmarks_snapshot` & `_benchmarks_snapshot_migrate` are hard-coded into the script, it seems better to be here rather than having a separated script file.  How about that?

Reply:
https://github.com/planetarium/lib9c/pull/329/commits/3c17c3439ba85fab2aeebe35073733023cebe298 Merged separated file into main.yml